app/uk/gov/hmrc/exports/controllers/request/ExportsDeclarationRequest.scala

```diff
@@ -66,138 +62,9 @@ case class ExportsDeclarationRequest(
-      updatedDateTime,
-      sourceId,
-      `type`,
-      dispatchLocation,
-      additionalDeclarationType,
-      consignmentReferences,
-      departureTransport,
-      borderTransport,
-      transportInformation,
-      parties,
-      locations,
-      items,
-      totalNumberOfItems,
-      previousDocuments,
-      natureOfTransaction
-    )
-  }
-
   import play.api.libs.functional.syntax._
   import play.api.libs.json._
 
-  val readsVersion1: Reads[ExportsDeclarationRequest] = (
-    (__ \ "createdDateTime").read[Instant] and
-      (__ \ "updatedDateTime").read[Instant] and
-      (__ \ "sourceId").readNullable[String] and
-      (__ \ "type").read[DeclarationType.Value] and
-      (__ \ "dispatchLocation").readNullable[DispatchLocation] and
-      (__ \ "additionalDeclarationType").readNullable[AdditionalDeclarationType.Value] and
-      (__ \ "consignmentReferences").readNullable[ConsignmentReferences] and
-      (__ \ "departureTransport").readNullable[DepartureTransport] and
-      (__ \ "borderTransport").readNullable[BorderTransport] and
-      (__ \ "transportInformation").readNullable[TransportInformation] and
-      (__ \ "parties").read[Parties] and
-      (__ \ "locations").read[Locations] and
-      (__ \ "items").read[Set[ExportItem]] and
-      (__ \ "totalNumberOfItems").readNullable[TotalNumberOfItems] and
-      (__ \ "previousDocuments").readNullable[PreviousDocuments] and
-      (__ \ "natureOfTransaction").readNullable[NatureOfTransaction]
-  ).apply(ExportsDeclarationRequest.version1 _)
-
-  val readsVersion2: Reads[ExportsDeclarationRequest] = (
-    (__ \ "createdDateTime").read[Instant] and
-      (__ \ "updatedDateTime").read[Instant] and
-      (__ \ "sourceId").readNullable[String] and
-      (__ \ "type").read[DeclarationType.Value] and
-      (__ \ "dispatchLocation").readNullable[DispatchLocation] and
-      (__ \ "additionalDeclarationType").readNullable[AdditionalDeclarationType.Value] and
-      (__ \ "consignmentReferences").readNullable[ConsignmentReferences] and
-      (__ \ "transport").read[Transport] and
-      (__ \ "parties").read[Parties] and
-      (__ \ "locations").read[Locations] and
-      (__ \ "items").read[Set[ExportItem]] and
-      (__ \ "totalNumberOfItems").readNullable[TotalNumberOfItems] and
-      (__ \ "previousDocuments").readNullable[PreviousDocuments] and
-      (__ \ "natureOfTransaction").readNullable[NatureOfTransaction]
-  ).apply(ExportsDeclarationRequest.version2 _)
-
-  val bothReads: Reads[ExportsDeclarationRequest] = (__ \ "transport").readNullable[Transport].flatMap[ExportsDeclarationRequest] {
-    case Some(_) => readsVersion2
-    case None    => readsVersion1
-  }
-
   implicit val format
-    : OFormat[ExportsDeclarationRequest] = OFormat(bothReads, Json.writes[ExportsDeclarationRequest]) // writes are used only for logging
+    : OFormat[ExportsDeclarationRequest] = Json.format[ExportsDeclarationRequest] // writes are used only for logging
```

Comment:
You can remove comment. It's not valid right now and there is no value to keep it

Reply:
It is. Changes in this writes though structure does not have effect on returned value by service.